std_msgs/Header header

string[] jointStateName
sensor_msgs/JointState[] jointState

string[] jacobianName
yumi_controller/Jacobian[] jacobian

string[] poseName
geometry_msgs/Pose[] pose
geometry_msgs/Twist[] poseTwist
geometry_msgs/Wrench[] poseWrench
